MAINTAINERS_GUIDE.md

```diff
@@ -0,0 +1,40 @@
+# Maintainers Guide
+
+## Responsibilities
+
+### Work on Current Issues and Create New Issues
+
+* Maintainers can choose any issues to work on and should make new issues when needed. Maintainers should add issues that they are working on to the the `In Progress` column of this [roadmap board](https://github.com/opensearch-project/opensearch-benchmark/projects/1). Any issue added to the `In Progress` column should be properly labeled. For more information on how to properly label issues and PRs, see the [labels](#labels) section of this document.
```

Comment:
Maintainers can select any issues they would like to work on and should add should add them to the `In Progress` column of the [roadmap board](https://github.com/opensearch-project/opensearch-benchmark/projects/1).  Any issue added to the `In Progress` column should be properly labeled. For more information on how to properly label issues and PRs, see the [labels](#labels) section of this document.

Maintainers should create new issues as needed.